zcash_client_backend/src/scanning.rs

```diff
@@ -905,8 +904,12 @@ mod tests {
     #[test]
     fn scan_block_with_txs_after_my_tx() {
         fn go(scan_multithreaded: bool) {
-            let account = AccountId::ZERO;
-            let extsk = ExtendedSpendingKey::master(&[]);
+            let seed = &[];
+            let account = AccountId::Zip32 {
+                seed_id: HDSeedFingerprint::from_seed(&SecretVec::new(seed.to_vec())),
+                account_id: zip32::AccountId::ZERO,
+            };
```

Comment:
Since this is used in a number of places, we should probably create a helper function for it.